source/scss/modules/_sections.scss

```diff
@@ -66,12 +66,12 @@
         color: $color-white;
 
         .jumbotron {
-            background-color: $color-green;
+            background-image: linear-gradient( $color-green, $--darker-secondary-color);
```

Comment:
I suggest making jumbotron a mixin or function that you can use anywhere it is needed so as not to repeat the same code.